src/pages/api/multipleProductValidity.ts

```diff
@@ -23,6 +24,10 @@ interface DecisionData {
     selectedServices?: ServicesInfo[];
     zoneName?: string;
     stops?: Stop[];
+    passengerType: string;
```

Comment:
If we have this interface extend Passenger Details, it will inherit all the passenger details for free rather than declare them here.